.azure/pipelines/components-e2e-tests.yml

```diff
@@ -58,11 +58,25 @@ jobs:
       displayName: Build JS
     - script: .dotnet/dotnet build ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-restore
       displayName: Build
-    - script: .dotnet/dotnet test ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-build --filter 'Quarantined!=true|Quarantined=false'
-                 -p:VsTestUseMSBuildOutput=false
-                 --logger:"trx%3BLogFileName=Microsoft.AspNetCore.Components.E2ETests.trx"
-                 --logger:"html%3BLogFileName=Microsoft.AspNetCore.Components.E2ETests.html"
-                 --results-directory $(Build.SourcesDirectory)/artifacts/TestResults/$(BuildConfiguration)/Unquarantined
+    - script: |
```

Comment:
is this running per project or for the whole test suite of E2E tests? 
I am thinking about parsing the amount of tests discovered, because if our test suite consists of lets say 100k tests, and we discovered 3k, then `"No test matches the given testcase filter"` will not be written; however there clearly would be a regression

Reply:
ye, I was thinking to set at least like 1k, so that we detect some of such situations. But we would need to raise it at some moment probably, so not sure about my idea.